python/japronto/server.py

```diff
@@ -18,4 +20,4 @@ def get_user(request):
 app.router.add_route('/user', create_user, 'POST')
 app.router.add_route('/user/{id}', get_user, 'GET')
 
-app.run(host='0.0.0.0', port=3000)
+app.run(host='0.0.0.0', worker_num=int(sys.argv[2]), port=3000)
```

Comment:
`sys.argv[2]` does not exists

Reply:
fixed, sorry, I made the change directly in the GitHub interface ... :P